applications/tari_base_node/src/parser.rs

```diff
@@ -275,7 +273,7 @@ impl Parser {
                 self.process_cancel_transaction(args);
             },
             SendTari => {
-                self.process_send_tari(args);
+                Self::process_send_tari(args, self.executor.clone(), self.wallet_transaction_service.clone());
```

Comment:
I think you did this because you were using `self` in the spawned task (`&'a self` where 'a does not meet the 'static lifetime requirement of `spawn`) 

```suggestion
                self.process_send_tari(args);
```

The rest of the modification follows

Reply:
Thank you for the comments. As discussed, this was a secondary change due to `self.process_send_tari` being called lower down in a spawned task inside a loop. Will change the interface to be consistent again and create a separate async function that can be called by both `process_send_tari` and `process_make_it_rain`, similar to @SWvheerden's suggestion.